src/app/features/collection/components/cards-list/cards-list.component.html

```diff
@@ -1,3 +1,14 @@
+@if (canMarkAllAsCollected()) {
+  <div class="tw-flex tw-justify-end tw-pt-1">
```

Comment:
it will be also to not display button, until collection info will be loaded and use skeleton for button during collection info loading state

Reply:
we have global skeleton for this page, so the content won't be seen until content loads. Should I add loader anyway?

Reply:
yea, it will be nice to have skeleton for initial loading (collectionInfoLoading) to improve UX

Reply:
added skeleton loader for button.